contrib/mod_snmp/mod_snmp.c

```diff
@@ -4013,6 +4026,8 @@ static int snmp_init(void) {
   snmp_pool = make_sub_pool(permanent_pool);
   pr_pool_tag(snmp_pool, MOD_SNMP_VERSION);
 
+  pr_event_register(&snmp_module, "core.max-instances",
```

Comment:
This listener needs to be in the master/daemon process, in order to properly see the "MaxInstances reached" events.